.github/workflows/deploy.yml

```
@@ -1,39 +1,42 @@
-name: Deploy selfhost
+name: "Deploy selfhost (backend)"
+
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)